### PR TITLE
Small fixes for graph related things

### PR DESCRIPTION
**Pull request by vlowingkloude**
## Summary

This PR contains some small fixes for graph related things.

### Bug Fixes

* Now single quotes in a string will be escaped 
* Added long literal support for Cypher, this is useful for ids.
* @gartens fixed incorrect split of cypher statements
* @datomo fixed label matches
* Similar to Neo4j, now when the data tab in the web UI is clicked the number of returned graph elements is limited to 25 to avoid transferring too much data

### CODE REVIEW

File: plugins/cypher-language/src/main/java/org/polypheny/db/cypher/CypherProcessor.java [hunk 1]
```diff
@@ -152,7 +154,79 @@ public AlgDataType getParameterRowType( Node left ) {
 
     @Override
     public List<String> splitStatements( String statements ) {
-        return Arrays.stream( statements.split( ";" ) ).filter( q -> !q.trim().isEmpty() ).toList();
```
Comment (gartens) [inline plugins/cypher-language/src/main/java/org/polypheny/db/cypher/CypherProcessor.java:155]: Actually `CypherParser.jj` supports multiple statements, (`CypherParser::parseStmts` even returns a list of `CypherStatement`s), so this should somehow be used directly if possible.

File: plugins/neo4j-adapter/src/main/java/org/polypheny/db/adapter/neo4j/util/NeoStatements.java [hunk 2]
```diff
@@ -538,7 +538,15 @@ static LiteralStatement literal_( PolyValue value ) {
...
 
     static LiteralStatement string_( PolyValue value ) {
-        return new LiteralStatement( value == null || value.isNull() ? null : "'" + value + "'" );
+        // TODO: do we need to implement more sophisticated way to escape strings?
+        if ( value == null || value.isNull()) {
+            return new LiteralStatement( null );
+        }
+        String quote = "'";
+        if ( value.toString().contains( "'" ) ) {
+            return new LiteralStatement( "\"" + value + "\"" );
```
Comment (gartens) [inline plugins/neo4j-adapter/src/main/java/org/polypheny/db/adapter/neo4j/util/NeoStatements.java:547]: What happens when values contain `"` and `'`?

Comment (vlowingkloude) [reply plugins/neo4j-adapter/src/main/java/org/polypheny/db/adapter/neo4j/util/NeoStatements.java:547]: Previously the generated bind code only uses single quotes to represent Cypher strings. Therefore, when a Cypher string contains single quotes, an exception will be triggered. For example, CREATE (:N {msg: "I'd like some tea"}).

Comment (vlowingkloude) [reply plugins/neo4j-adapter/src/main/java/org/polypheny/db/adapter/neo4j/util/NeoStatements.java:547]: I didn't really consider the mix of quotes in a string. Will do a fix now and add some test cases

**Pull request closed without merging**